_docs/tasks/traffic-management-v1alpha3/ingress.md

```diff
@@ -74,30 +67,17 @@ using Istio routing rules, exactly in the same was as for internal service reque
       name: httpbin-gateway
     spec:
       selector:
-        istio: ingressgateway # use istio default controller
+        istio: ingressgateway # use Istio default gateway implementation
       servers:
       - port:
           number: 80
           name: http
           protocol: HTTP
         hosts:
-        - "*"
-      - port:
-          number: 443
-          name: https
-          protocol: HTTPS
-        tls:
-          mode: SIMPLE
-          serverCertificate: /tmp/tls.crt
-          privateKey: /tmp/tls.key
-        hosts:
-        - "*"
+        - "foo.bar.com"
```

Comment:
How about something more consistent with the purpose, e.g., "myhttpbin.com" or "httpbin.example.com"